drivers/gles3/storage/config.cpp

```diff
@@ -88,11 +88,11 @@ Config::Config() {
 	rgtc_supported = extensions.has("GL_EXT_texture_compression_rgtc") || extensions.has("GL_ARB_texture_compression_rgtc") || extensions.has("EXT_texture_compression_rgtc");
 #endif
 
-	glGetIntegerv(GL_MAX_VERTEX_TEXTURE_IMAGE_UNITS, &max_vertex_texture_image_units);
-	glGetIntegerv(GL_MAX_TEXTURE_IMAGE_UNITS, &max_texture_image_units);
-	glGetIntegerv(GL_MAX_TEXTURE_SIZE, &max_texture_size);
-	glGetIntegerv(GL_MAX_UNIFORM_BLOCK_SIZE, &max_uniform_buffer_size);
-	glGetIntegerv(GL_MAX_VIEWPORT_DIMS, max_viewport_size);
+	glGetInteger64v(GL_MAX_VERTEX_TEXTURE_IMAGE_UNITS, &max_vertex_texture_image_units);
+	glGetInteger64v(GL_MAX_TEXTURE_IMAGE_UNITS, &max_texture_image_units);
```

Comment:
There's another one `glGetIntegerv(GL_MAX_TEXTURE_IMAGE_UNITS, ...)` in the `shader_gles3.cpp`, not related to the issue, but probably should be changed to 64-bit version as well for consistency.

https://github.com/godotengine/godot/blob/6758a7f8c07d1f4c8ec4f052ded6d26402967ebe/drivers/gles3/shader_gles3.cpp#L774

Reply:
Done!